outgoing_email_by_model/tests/test_mailserver_by_model.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo_test_helper import FakeModelLoader
+
+from odoo.tests import Form
+
+from odoo.addons.base.tests.common import BaseCommon
+
+
+class TestMailserverByModel(BaseCommon):
+    at_install = False
+    post_install = True
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Comment:
```suggestion
```

Sorry, I should have been more complete in my previous review.
`BaseCommon` already set `tracking_disable` to `True`, among other parameters:
```python
DISABLED_MAIL_CONTEXT = {
    'tracking_disable': True,
    'mail_create_nolog': True,
    'mail_create_nosubscribe': True,
    'mail_notrack': True,
    'no_reset_password': True,
}
```

cf. https://github.com/odoo/odoo/blob/17.0/odoo/addons/base/tests/common.py#L12-L35